lib/hippie/client.js

```diff
@@ -13,6 +13,7 @@ var http = require('http');
 var serializers = require('./serializers');
 var parsers = require('./parsers');
 var expect = require('./expect');
+var Promise = require('es6-promise').Promise;
```

Comment:
ping @vesln I have some time over the next few days to knock this out if you let me know which approach you prefer